rest-service/manager_rest/rest/resources_v1/status.py

```diff
@@ -78,32 +78,34 @@ def get(self, **kwargs):
 
         return {'status': 'running', 'services': jobs}
 
-    def _should_be_in_services_output(self, job):
-        if job['unit_id'] not in OPTIONAL_SERVICES:
-            return True
 
-        if job['instances']:
-            # We have some details of the systemd job...
-            if job['instances'][0]['LoadState'] != 'not-found':
-                # And since its LoadState wasn't not-found, it should be
-                # installed and working
-                return True
+def should_be_in_services_output(job, optional_services=OPTIONAL_SERVICES):
+    if job['unit_id'] not in optional_services:
+        return True
 
-        # If we reach here then the service is optional and not installed
-        return False
+    if job['instances']:
+        # We have some details of the systemd job...
+        if job['instances'][0]['LoadState'] != 'not-found':
+            # And since its LoadState wasn't not-found, it should be
+            # installed and working
+            return True
+
+    # If we reach here then the service is optional and not installed
+    return False
 
-    def _get_systemd_manager_services(self):
-        """Services the status of which we keep track of.
 
-        :return: a dict of {service_name: label}
-        """
-        services = {}
+def get_systemd_manager_services(base_services=BASE_SERVICES,
```

Comment:
We shouldn't be setting default arg values to mutable values. At the moment, we don't break it, but it's generally bad practise as it can lead to unexpected interesting situations (TM) in future.